npm-debug.log

```diff
@@ -0,0 +1,32 @@
+0 info it worked if it ends with ok
```

Comment:
npm-debug.log is an artifact that you should check into git or GitHub. It's useful for you during development, but will not be useful to anyone else using your code. Delete it from your GitHub repo.